README.md

```diff
@@ -1,25 +1,46 @@
-# fwupd
 
-[![Build Status](https://github.com/fwupd/fwupd/actions/workflows/main.yml/badge.svg)](https://github.com/fwupd/fwupd/actions/workflows/main.yml)
-[![CodeQL](https://github.com/fwupd/fwupd/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/fwupd/fwupd/actions/workflows/codeql-analysis.yml)
-[![Coverity Scan Build Status](https://scan.coverity.com/projects/10744/badge.svg)](https://scan.coverity.com/projects/10744)
-[![Fuzzing Status](https://oss-fuzz-build-logs.storage.googleapis.com/badges/fwupd.svg)](https://bugs.chromium.org/p/oss-fuzz/issues/list?sort=-opened&can=1&q=proj:fwupd)
-[![CircleCI](https://circleci.com/gh/fwupd/fwupd/tree/main.svg?style=svg)](https://circleci.com/gh/fwupd/fwupd/tree/main)
+[![Badge License]][License]
 
-This project aims to make updating firmware on Linux automatic, safe and reliable.
+<div align = center>
 
-Additional information is available [at the website](https://fwupd.org/).
+<br>
+  
+# FWUPD
+
+***Automatic*** *, **Reliable** & **Safe** updating of* ***Linux Firmware***
+
+<br>
+
+[![Badge Website]][Website] 
+[![Badge Matrix]][Matrix] 
+[![Badge IRC]][IRC]
+
+<br>
+
+---
+
+[![Badge Build]][Build Status] 
+[![Badge CodeQL]][CodeQL] 
+[![Badge Coverity]][Coverity] 
+[![Badge Fuzzing]][Fuzzing] 
+[![Badge CircleCI]][CircleCI]
+
+---
+
+<br>
+
+</div>
 
 ## Where to get help?
 
-- GitHub issues & discussions in [this repository](https://github.com/fwupd/fwupd)
+- GitHub issues & discussions in [this repository][Repository]
 - Libera IRC channel `#fwupd`.
-  You can join through Libera's [IRC](https://libera.chat/)
-  or via the IRC bridge on [Matrix](https://matrix.to/#/#fwupd:libera.chat).
+  You can join through Libera's [IRC]
+  or via the IRC bridge on [Matrix].
 
 ## Compiling
 
-The most up to date compilation instructions are available in the [Wiki](https://github.com/fwupd/fwupd/wiki/Compilation).
+The most up to date compilation instructions are available in the [Wiki][Compiling].
```

Comment:
I'm not really a fan of the detached links; it makes it a bit hard to follow when using it as a text file imho. @superm1 your opinion?

Reply:
On that topic, I have a **[little something]**.

[little something]: https://github.com/MarkedDown/Links

Reply:
I personally open markdown in vscode with markdown preview all the time. So I am indifferent.

Reply:
To be honest, I'm not a fan of the detached links; I edit and view most of the markdown files in a plain text editor, and having to scroll down the file to find a link doesn't really work for me. I don't agree the indirection cleans up the file, especially when most of the links are `[link description][link anchor]` and the `link anchor` is often as long as the original URL was.